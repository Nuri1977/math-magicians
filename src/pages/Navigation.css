header {
  background-color: var(--primary);
  height: 62px;
  color: var(--terciary);
}

.logo {
  padding: 5px 20px;
  color: var(--terciary);
}

li * {
  padding: 20px;
  color: var(--terciary);
  font-size: 1.5rem;
}
