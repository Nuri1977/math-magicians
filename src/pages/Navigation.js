import React from 'react';
import { Link } from 'react-router-dom';
import './Navigation.css';

export default function Navigation() {
  return (
    <header>
      <div className="container flex">
        <h1>
          <Link to="/" className="logo">Math Magicians</Link>
        </h1>
        <nav>
          <ul className="flex">
            <li>
              <Link to="/">Home</Link>
            </li>
            <li>
              <Link to="/calculator">Calculator</Link>
            </li>
            <li>
              <Link to="/quote">Quote</Link>
            </li>
          </ul>
        </nav>
      </div>

    </header>

  );
}
