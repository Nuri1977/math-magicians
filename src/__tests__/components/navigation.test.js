import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navigation from '../../pages/Navigation';

describe('Testing home component', () => {
  it('Home renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Navigation />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
