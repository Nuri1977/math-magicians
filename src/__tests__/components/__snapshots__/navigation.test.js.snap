// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing home component Home renders correctly 1`] = `
<header>
  <div
    className="container flex"
  >
    <h1>
      <a
        className="logo"
        href="/"
        onClick={[Function]}
      >
        Math Magicians
      </a>
    </h1>
    <nav>
      <ul
        className="flex"
      >
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
